src/os/inc/osapi-os-core.h

```diff
@@ -138,6 +138,21 @@ typedef struct
    uint8 object_ids[(OS_MAX_NUM_OPEN_FILES + 7) / 8];
 } OS_FdSet;
 
+/**
+ * @brief For the OS_SelectSingle() function's in/out StateFlags parameter,
+ * the state(s) of the stream and the result of the select is a combination
+ * of one or more of these states.
+ *
+ * @sa OS_SelectSingle()
+ */
+typedef enum
+{
+    OS_STREAM_STATE_BOUND      = 0x01,
```

Comment:
`    OS_STREAM_STATE_BOUND      = 0x01, /**< @brief Stream bound state */` and similar for the other three.

I realize the comment isn't really additional useful information, but will avoid "undocumented element" warnings in the future.